linker/Tests/TestCasesRunner/TestCaseMetadaProvider.cs

```diff
@@ -106,9 +106,13 @@ public virtual IEnumerable<SetupCompileInfo> GetSetupCompileAssembliesAfter ()
 
 		public virtual IEnumerable<string> GetDefines ()
 		{
-			return _testCaseTypeDefinition.CustomAttributes
-				.Where (attr => attr.AttributeType.Name == nameof (DefineAttribute))
-				.Select (attr => (string) attr.ConstructorArguments.First ().Value);
+			// There are a few tests related to native pdbs where the assertions are different between windows and non-windows
+			// To enable test cases to define different expected behavior we set this special define
+			if (Environment.OSVersion.Platform == PlatformID.Win32NT)
+				yield return "TEST_FW_DEFINED_WINDOWS";
```

Comment:
Could you simplify the name? I think something like `WIN32` should be more than enough

Reply:
Done